include/vcpkg/base/messages.h

```diff
@@ -1933,6 +1915,28 @@ namespace vcpkg
                     "The name 'search' is the name of a command that is not localized.",
                     "No packages are installed. Did you mean `search`?");
     DECLARE_MESSAGE(NoLocalizationForMessages, (), "", "No localized messages for the following: ");
+    DECLARE_MESSAGE(NonExactlyArgs,
+                    (msg::command_name, msg::expected, msg::actual),
+                    "{expected} and {actual} are integers",
+                    "the command '{command_name}' requires exactly {expected} arguments, but {actual} were provided");
```

Comment:
Should `the` be capitalized? Should `arguments` be `argument(s)`?

Reply:
>Should the be capitalized?

I don't think so; at least, that doesn't seem to be the usual convention after `error: `

>Should arguments be argument(s)?

No, there's a different message for 1 argument. https://github.com/microsoft/vcpkg-tool/pull/864/files#diff-606f2be8ae8aaaf2671f5952c61466ffece81d1c7e7ae00ee185af78f2af264dR930